service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -50,7 +50,8 @@ public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDo
         Bundle bundle = new Bundle();
         bundle.setType(TRANSACTION);
 
-        Encounter encounter = encounterMapper.mapEncounter(clinicalDocument);
+        List<HealthcareService> healthcareServiceList = healthcareServiceMapper.mapHealthcareService(clinicalDocument);
```

Comment:
Why did you decide to move HealthcareService here instead of leaving it as part of ReferralRequestMapping?

Reply:
For any future resource that may use it in the future, no duplicates will be created. Also keeps it consistent with rest of code.